### PR TITLE
[Learn Design][Designcember] Adding the next five modules

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes: #6409 
Fixes: #6410 
Fixes: #6411 
Fixes: #6412 
Fixes: #6413 

Adds the next five articles. To be merged as part of Designcember.

### CODE REVIEW

**Review by adactio — APPROVED** [review]
Looks good!

(I made some small formatting adjustments, as mentioned in chat.)

**Review by argyleink — APPROVED** [review]
nice, looks good. new page styles are nice too 👍🏻

File: src/site/content/en/learn/design/icons/index.md [hunk 1]
```diff
@@ -4,8 +4,249 @@ description: >
...
+
+```html
+<img src="image.svg" alt="A description of the image." width="25" height="25">
+<img src="image.svg" alt="A description of the image." width="250" height="250">
+```
+
+{% Codepen {
+ user: 'web-dot-dev',
+ id: 'YzrKoWY',
+ height: 300,
```
Comment (camden) [inline src/site/content/en/learn/design/icons/index.md:31]:
```suggestion
 height: 360,
```

File: src/site/content/en/learn/design/theming/index.md [hunk 2]
```diff
@@ -4,8 +4,343 @@ description: >
...
+button {
+  background-color: var(--ink-color);
+  color: var(--page-color);
+}
+```
+
+{% Codepen {
+ user: 'web-dot-dev',
+ id: 'qBPWzrj',
+ height: 400,
```
Comment (camden) [inline src/site/content/en/learn/design/theming/index.md:174]:
```suggestion
 height: 440,
```

**Review by camden — CHANGES_REQUESTED** [review]
Updated the heights of a couple of demos to avoid scrollbars.

File: src/site/content/en/learn/design/icons/icons.assess.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+questions:
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "0"
+    stem: "SVG can handle any pixel density with one single file or `<svg>` code block."
+    options:
+      - content: "True"
+        rationale: "SVG includes how to draw the shapes and lines, at any pixel density or scale or zoom."
+      - content: "False"
+        rationale: "Unlight .png or .jpg, SVG has no need for `srcset` or a `<picture>` element."
```
Comment (rachelandrew) [inline src/site/content/en/learn/design/icons/icons.assess.yml:13]:
```suggestion
        rationale: "Unlight .png or .jpg, SVG has no need for `srcset` or a `<picture>` element."
```
```suggestion
        rationale: "Unlike `.png` or `.jpg`, SVG has no need for `srcset` or a `<picture>` element."
```

File: src/site/content/en/learn/design/icons/icons.assess.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      - content: "Styleable from CSS"
+        rationale: "Match SVG icon shapes to buttons and brand colors."
+      - content: "No download required"
+        rationale: "All the instructions are there."
+      - content: "Lazy loaded by default"
+        rationale: "No download to be lazy about."
+      - content: "Less CPU usage"
+        rationale: "I made this up."
+      - content: "Light or dark themed without a new asset"
+        rationale: "Media queries can change inline SVG styles yep!"
```
Comment (rachelandrew) [inline src/site/content/en/learn/design/icons/icons.assess.yml:28]:
```suggestion
      - content: "Styleable from CSS"
        rationale: "Match SVG icon shapes to buttons and brand colors."
      - content: "No download required."
        rationale: "All the instructions are there."
      - content: "Lazy loaded by default."
        rationale: "No download to be lazy about."
      - content: "Less CPU usage."
        rationale: "I made this up."
      - content: "Light or dark themed without a new asset."
        rationale: "Media queries can change inline SVG styles!"
```

File: src/site/content/en/learn/design/theming/theming.assess.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    cardinality: "1+"
+    correctAnswers: "0,1,2"
+    stem: "So you support dark theme, but all the form inputs are still light themed. What can you do?"
+    options:
+      - content: "Add `html { color-scheme: light dark; }` to your CSS."
+        rationale: "This signals from CSS that the form inputs should match the system color scheme."
+      - content: 'Add `<meta name="supported-color-schemes" content="light dark">` to your HTML `<head>` tag.'
+        rationale: "This signals from HTML that the form inputs should match the system color scheme."
+      - content: "Write a bunch of CSS to change all the defaults of the input"
+        rationale: "This work too, but is a bit harder."
```
Comment (rachelandrew) [inline src/site/content/en/learn/design/theming/theming.assess.yml:37]:
```suggestion
      - content: "Write a bunch of CSS to change all the defaults of the input."
        rationale: "This works too, but is a bit harder."
```

File: src/site/content/en/learn/design/typography/typography.assess.yml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    correctAnswers: "2"
+    stem: "`clamp()` is useful for fluid typography because"
+    options:
+      - content: "It allows easy embedding of `calc()` functions"
+        rationale: "While this is true, it's not a good reason to use `clamp()` for typography."
+      - content: "The browser support for it is great."
+        rationale: "While this is true, it's not a good reason to use `clamp()` for typography."
+      - content: "It allows locking the font size between sensible minimums and maximums while also providing a scalable middle value."
+        rationale: "Exactly, prevent too small or too large text, while also providing a smooth scaling font size."
+      - content: "It makes the math easy"
```
Comment (rachelandrew) [inline src/site/content/en/learn/design/typography/typography.assess.yml:25]:
```suggestion
      - content: "It makes the math easy."
```

File: src/site/content/en/learn/design/typography/typography.assess.yml [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+        rationale: "Try again."
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "2"
+    stem: "Which type of `line-height` values were recommended in this guide?"
+    options:
+      - content: "`24px`"
+        rationale: "The post explicitly says don't use pixel values for `line-height`"
```
Comment (rachelandrew) [inline src/site/content/en/learn/design/typography/typography.assess.yml:33]:
```suggestion
        rationale: "The post explicitly says don't use pixel values for `line-height`."
```

File: src/site/content/en/learn/design/typography/typography.assess.yml [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    cardinality: "1+"
+    correctAnswers: "0,3"
+    stem: "What does `font-display` do?"
+    options:
+      - content: "Tells the browser how to manage switching from a system font to a web font."
+        rationale: "Helps with the transition to a custom font."
+      - content: "Allows setting the font to `block` or `inline-block`."
+        rationale: "Fonts do not have display types."
+      - content: "Changes if the font is hidden or not."
+        rationale: "Fonts cannot be hidden"
```
Comment (rachelandrew) [inline src/site/content/en/learn/design/typography/typography.assess.yml:50]:
```suggestion
        rationale: "Fonts cannot be hidden."
```

**Pull request merged**